torch/package/_package_unpickler.py

```diff
@@ -23,4 +30,12 @@ def find_class(self, module, name):
             elif module in _compat_pickle.IMPORT_MAPPING:
                 module = _compat_pickle.IMPORT_MAPPING[module]
         mod = self._importer.import_module(module)
+
+        if hasattr(self._importer, "selective_extern_packages"):
```

Comment:
What's going on here? This should be handled internally by `self._importer.import_module`.

Reply:
this is to appease the type checker. I assume for these cases make them asserts.

Reply:
If you're talking about the general changes to the unpickler. Unpickling happens to be a case where unfortunately the overriden __getattr__ function of the module isn't called. Therefore, we have figure out if we're using the original or shimmed module here.

Reply:
Actually after retesting it seems that these changes can / should be reverted.